### ISSUE TITLE
S3: comparison operation with <= and >= should be =< and =>

### ISSUE THREAD

**Issue by kgracey** [parallaxinc/BlocklyProp#671]
Project is http://demo.blockly.parallax.com/blockly/editor/blocklyspin.jsp?project=347#

```
'SERIAL_TERMINAL USED
CON
  _clkmode = xtal1 + pll16x
  _xinfreq = 5_000_000

VAR
  long leftLineSensor
  long rightLineSensor

OBJ
  scribbler    : "Block_Wrapper"
  serial    : "Parallax Serial Terminal"

PUB Setup
  Scribbler.Start
  Scribbler.SerialStart( 9600 )
  Start


PUB Start

  repeat
    leftLineSensor := Scribbler.LineSensor(Scribbler#LEFT)
    rightLineSensor := Scribbler.LineSensor(Scribbler#RIGHT)
    Scribbler.SerialPositionX(0)
    Scribbler.SerialPositionY(1)
    Scribbler.SerialStr(String("Left ="))
    Scribbler.SerialPositionX(10)
    Scribbler.SerialPositionY(1)
    Scribbler.SerialDec(leftLineSensor)
    Scribbler.SerialPositionX(0)
    Scribbler.SerialPositionY(2)
    Scribbler.SerialStr(String("Right = "))
    Scribbler.SerialPositionX(10)
    Scribbler.SerialPositionY(2)
    Scribbler.SerialDec(rightLineSensor)
    if leftLineSensor <= 45 AND rightLineSensor <= 45
      Scribbler.SetLED(Scribbler#LEFT, Scribbler#COLOR_ff0000)
      Scribbler.SetLED(Scribbler#CENTER, Scribbler#COLOR_00ff00)
      Scribbler.SetLED(Scribbler#RIGHT, Scribbler#COLOR_ff0000)
      Scribbler.MotorSet(50, 50, 0)
    elseif leftLineSensor <= 45 AND rightLineSensor > 45
      Scribbler.SetLED(Scribbler#LEFT, Scribbler#COLOR_ff0000)
      Scribbler.SetLED(Scribbler#CENTER, Scribbler#COLOR_00ff00)
      Scribbler.SetLED(Scribbler#RIGHT, Scribbler#COLOR_00ff00)
      Scribbler.MotorSet((-30), 50, 0)
    elseif leftLineSensor > 45 AND rightLineSensor < 45
      Scribbler.SetLED(Scribbler#LEFT, Scribbler#COLOR_00ff00)
      Scribbler.SetLED(Scribbler#CENTER, Scribbler#COLOR_00ff00)
      Scribbler.SetLED(Scribbler#RIGHT, Scribbler#COLOR_ff0000)
      Scribbler.MotorSet(50, (-30), 0)
    else
      Scribbler.SetLED(Scribbler#LEFT, Scribbler#COLOR_000000)
      Scribbler.SetLED(Scribbler#CENTER, Scribbler#COLOR_000000)
      Scribbler.SetLED(Scribbler#RIGHT, Scribbler#COLOR_000000)
```

**Comment by dcarrier-parallax** [comment]
Addressed in PR #672; please verify.

**Comment by kgracey** [comment]
Verified and fixed!
